lib/pbench/test/unit/server/test_datasets_metadata.py

```diff
@@ -76,6 +67,18 @@ def test_get2(self, client, server_config, provide_metadata):
             "user": {"contact": "me@example.com"},
         }
 
+    def test_get_bad_query(self, client, server_config, provide_metadata):
+        response = client.get(
+            f"{server_config.rest_uri}/datasets/metadata",
+            query_string={
+                "name": "drb",
+                "controller": "foobar",
+                "metadata": "dashboard.seen,server.deletion,dataset.access,user",
+            },
+        )
+        assert response.status_code == HTTPStatus.BAD_REQUEST
+        assert response.json == {"message": "Unknown URL query keys: controller"}
```

Comment:
Is there a test which checks multiple unknown keys?

Reply:
Hmm; no, there isn't. This was just a result of discovering that the tests still used the obsolete `controller` key, and I figured that was worth testing.